tests/wicked/wlan/sut/t15_wpa3_personal.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test WiFi setup with wicked for wpa3 transition
+#          (WPA-PSK or SAE with DHCP)
+#   - The AP is connfiguered to allow WPA-PSK and SAE connections
+#   - Connect to AP with WPA-PSK
+#   - Connect to AP with WPA-PSK + PMF
+#   - Connect to AP with SAE
+#   - Connect to AP with SAE + PMF
+#   - Each connection is checked with data bi-directional traffic
+#
+# Maintainer: cfamullaconrad@suse.com
+
+
+use Mojo::Base 'wicked::wlan';
+use testapi;
+
+has wicked_version => '>=0.6.70';
+has need_key_mgmt => 'SAE';
+has ssid => 'Virtual WiFi SAE Secured';
+has psk => 'TopSecretWifiPassphrase!';
+
+has hostapd_conf => q(
```

Comment:
I really like this approach that basically test is come up just as bunch of config files :+1:  nice !

Reply:
thx